lib/alephant/preview/server.rb

```diff
@@ -32,16 +32,23 @@ class Server < Sinatra::Base
       end
 
       get "/preview/:id/:template/:region/?:fixture?" do
+        response["X-Sequence"] = Time.now.to_i
+
         render_preview
       end
 
       get "/component/:template/?:fixture?" do
         params["id"] = find_id_from_template params["template"]
         params["fixture"] = "responsive" unless params["fixture"]
+
+        response["X-Sequence"] = Time.now.to_i
```

Comment:
You could factor the `Time.now.to_i` into a `sequence_id` method?  Just in case there is a real thing we want to do with it in the future.